security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -578,27 +578,27 @@ func (sc *SecretCache) rotate(updateRootFlag bool) {
 			return true
 		}
 
+		if sc.configOptions.CredFetcher != nil {
+			// Refresh token through credential fetcher.
+			cacheLog.Infof("%s getting a new token through credential fetcher", logPrefix)
+			t, err := sc.configOptions.CredFetcher.GetPlatformCredential()
+			if err != nil {
+				cacheLog.Errorf("failed to get credential token: %v", err)
```

Comment:
cacheLog.Errorf("credential fetcher failed to get credential token: %v", err)

Reply:
Done.